BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/bank/bunq/BunqBank.java

```diff
@@ -162,6 +161,25 @@ private Void loadOrCreateClientKeys() {
         });
     }
 
+    @Override
+    public CompletableFuture<Boolean> sendTransaction(Transaction transaction) {
+        BunqTransaction tran = (BunqTransaction) transaction;
+        CompletableFuture<PaymentService.PostResponse> future;
+        PaymentService service;
+
+        service = retrofit.create(PaymentService.class);
+
+        future = service.createPayment(tran.convertToRequest(), tran.getAcount().getParty().getId(), tran.getAcount().getId());
+        return future.thenApply((PaymentService.PostResponse response) -> {
```

Comment:
Don't, as PostResponse is not a unique class name: it is an inner name of any API that has a post response object. It is also not clear when reading the code what "PostResponse" refers to: it is now.